src/controller/api/index.js

```diff
@@ -0,0 +1,75 @@
+const {
+  getNotesList,
+  validKeys,
+  writeNotesToFile,
+  identifyNoteId,
+} = require("../../utility/util");
+const { v4: uuidv4 } = require("uuid");
+
+const getNotes = (req, res) => {
+  // get notes data fom the db file by reading, then parse the string and then res.json
+
+  const notesList = getNotesList();
+
+  res.json(notesList);
+};
+
+const addNewNote = (req, res) => {
+  const payLoad = req.body;
+
+  const isValid = validKeys.every((each) => {
```

Comment:
Would be helpful to have the `validKeys` array declared here rather than imported from elsewhere. Since it is only used once, there is no need extract it, and it would help to be able to see quickly here what is being checked.